### PR TITLE
Add information reg. root & SRS feature to FAQ entry `rooted_devices`

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description
This PR adds the information, that the new SRS feature won't work on rooted devices, to the FAQ entry `rooted_devices`.

## What was changed?
- https://www.coronawarn.app/de/faq/results/#rooted_devices
- https://www.coronawarn.app/en/faq/results/#rooted_devices

## Screenshot
Pff, that's outdated. Live preview available here:
https://cwa-website-pn8tnqv5g-ein-tim.vercel.app/de/faq/results/#rooted_devices  & https://cwa-website-pn8tnqv5g-ein-tim.vercel.app/en/faq/results/#rooted_devices

### CODE REVIEW

**Comment by Ein-Tim** [comment]
@MikeMcC399 Please take a look. I added the new text as an "update", but if you think it would be better to just add it in the normal text I can also do that!

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Regarding "Warning without TAN" and SafetyNet `#rooted_devices`:

1. The additional text does not connect warning without a TAN to the SafetyNet feature. It would be better if it did so.
2. In the German text "Sie ihr Gerät" doesn't make sense. It should be "Sie Ihr Gerät". This wording may not be necessary if the additional text is reworked.

Regarding `#update_faq`:

1. When the English date is written using ordinal extensions, these depend on the number of the date, and the abbreviation is the last two letters of the ordinal. 1st, 2nd, 3rd, 4th, etc. "21th" is wrong. It needs to be "21st". The CWA blogs all write the date without the letters after the numerals i.e. "Jan 21, 2023", not "Jan 21st, 2023". This is simpler. The FAQs are inconsistent in this respect.
2. I don't know how @larswmh envisioned this being handled, however it would need to be the date of publication (merge) not the date of the PR submission.
3. If there are multiple PRs open this could cause date conflicts.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I will add a sentence clarifying why warning without TAN doesn't work.

I won't touch `update_faq` until there is feedback from the team on how to proceed here!

File: src/data/faq.json [hunk 1]
```diff
@@ -1900,7 +1900,7 @@
                                 "active": false,
                                 "textblock": [
                                     "<b>Updated 21 January, 2023</b>",
-                                    "As of version 3.0 users of the Corona-Warn-App can warn others without a TAN. This will not work if you have rooted your device.",
+                                    "As of version 3.0 users of the Corona-Warn-App can warn others without a TAN. This will not work if you have rooted your device, as the SafetyNet check the app perofrms when attempting to warn others without a TAN fails.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1903]: "perofrms" => "performs"

Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`4d5e44c` (#3344)](https://github.com/corona-warn-app/cwa-website/pull/3344/commits/4d5e44caf9d536c634e63b4cd837575f70a993a4).

**Comment by dsarkar** [comment]
@Ein-Tim thanks for the PR. I would suggest to swap first and second paragraph.

File: src/data/faq.json [hunk 2]
```diff
@@ -2737,7 +2739,7 @@
                                 "anchor": "update_faq",
                                 "active": false,
                                 "textblock": [
-                                    "The last content changes to FAQ articles were made on January 20th, 2023.",
+                                    "The last content changes to FAQ articles were made on January 21th, 2023.",
```
Comment (MikeMcC399) [inline src/data/faq.json:2742]: use either 23rd or 23

File: src/data/faq_de.json [hunk 3]
```diff
@@ -2738,7 +2740,7 @@
                                 "anchor": "update_faq",
                                 "active": false,
                                 "textblock": [
-                                    "Die letzten inhaltlichen Änderungen an FAQ-Artikeln wurden am 20. Januar 2023 getätigt.",
+                                    "Die letzten inhaltlichen Änderungen an FAQ-Artikeln wurden am 21. Januar 2023 getätigt.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2743]: now 23.

**Comment by Ein-Tim** [comment]
@dsarkar 

Ok, would you leave the `Updated` out and just put it in the normal text?

Please also give advice on how to handle the updated date.

**Comment by dsarkar** [comment]
@Ein-Tim keep "Updated" with the date of  today. I will merge then later today. Will you also update  #update_faq ? Thanks.

**Comment by Ein-Tim** [comment]
@dsarkar 

> keep "Updated" with the date of today in the first line, in bold, as you have done already. I will merge then later today. Will you also update #update_faq ? Thanks.

I'll do all that! But what do you mean with changing the first and the second paragraph? Putting the information that data donation won't work before the information that warning others won't work?

File: src/data/faq_de.json (hunk 3 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Fixed with the most recent commit!

**Comment by Ein-Tim** [comment]
@dsarkar & @MikeMcC399 Please take a look at [corona-warn-app/cwa-website@`2d85115` (#3344)](https://github.com/corona-warn-app/cwa-website/pull/3344/commits/2d851157521b1518ad3fed439890f9e89f653b09)! I hope this is what you meant, @dsarkar? If not, let me know!

Live preview is available here: https://cwa-website-pn8tnqv5g-ein-tim.vercel.app/de/faq/results/#rooted_devices  & https://cwa-website-pn8tnqv5g-ein-tim.vercel.app/en/faq/results/#rooted_devices

**Comment by MikeMcC399** [comment]
@Ein-Tim 
For questions of style I defer to @dsarkar.
The date is now wrong again, since the PR was not merged yesterday. It is now Jan 24 (or 24th).

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks for your input here. We can standardise the English date later. I personally don't have any preference. @MikeMcC399 what is your preference?

**Comment by MikeMcC399** [comment]
@dsarkar 

I suggest to use the same format for English dates in FAQs as used in the blog, for example:

"January 24, 2023"

- The order month, day, year is generally used in American English, however, since it is unambiguous, it causes no problem for British English readers to correctly understand. 

- Avoiding the use of 1st, 2nd, 3rd, 4th, etc, makes it cleaner and assists non-native English speakers in writing dates correctly. It is also more modern to omit this.

**Comment by dsarkar** [comment]
@MikeMcC399 ok, thanks, sounds good. Let's change it then the to the 'simple' format.

**Comment by MikeMcC399** [comment]
- link to related Android issue https://github.com/corona-warn-app/cwa-app-android/issues/5813

**Pull request merged**